mucommander-core/src/main/java/com/mucommander/ui/action/impl/ToggleTerminalAction.java

```diff
@@ -90,9 +92,14 @@ public void performAction() {
 
                 terminal.getTerminalPanel().addCustomKeyListener(new KeyAdapter() {
                     public void keyPressed(KeyEvent keyEvent) {
-                        // TODO get the real configured key seq
-                        if (keyEvent.getKeyCode() == KeyEvent.VK_O &&
-                                (keyEvent.getModifiersEx() & KeyEvent.CTRL_DOWN_MASK) != 0) {
+                        KeyStroke pressedKeyStroke = KeyStroke.getKeyStrokeForEvent(keyEvent);
+                        KeyStroke keyStroke = ActionKeymap.getAccelerator(ActionType.ToggleTerminal.toString());
+                        if (pressedKeyStroke.equals(keyStroke)) {
```

Comment:
I would make it a one condition or with 'else' instead

Reply:
ack, changed it to a single condition